node_modules/socks/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "socks",
   "private": false,
-  "version": "2.8.3",
+  "version": "2.8.4",
```

Comment:
🤔 No other files under `socks` changed?

Reply:
Yep! There were several dependencies that look like they burned versions for things that didn't affect the published module.  A good example was `is-cidr` which had updates to virtually all its devDependencies, but no production ones nor any code.

Reply:
> updating to the latest version regardless.

This is exactly it.  This then clears that dependency from outdated checks.

Reply:
Yup, in socks' case it looks like it was a README update: https://github.com/JoshGlazebrook/socks/commits/master/